jenkins-integration/jenkins-jobs/job_bootstrap.groovy

```diff
@@ -0,0 +1,71 @@
+import groovy.io.FileType
+import java.nio.file.Paths
+
+// NOTE: these determine the default repo/branch that the seed job will
+// poll to look for Jenkinsfiles.  For production they should always
+// be set to the PL gplt repo's master branch.  For dev, you may want
+// to change them to point to your fork or your git daemon (though you can
+// accomplish the same by simply modifying the values in the Jenkins gui
+// on the 'configuration' screen for the seed job.
+
+def git_repo = 'https://github.com/puppetlabs/gatling-puppet-load-test.git'
```

Comment:
There's a few places where you might end up wanting to change repo name/branch name/environment name for doing development. It seems like it might be helpful to have a comment (like `CHANGE ME FOR DEVELOPMENT` or something better named/worded) that you could easily grep for to find all these places?